views/default/core/js/ui.js

```diff
@@ -20,8 +20,8 @@ elgg.ui.init = function () {
 		}
 	});
 
-	$('.elgg-system-messages li').animate({opacity: 0.9}, 6000);
-	$('.elgg-system-messages li.elgg-state-success').fadeOut('slow');
+	$('.elgg-page-default .elgg-system-messages .elgg-message').parent().animate({opacity: 0.9}, 6000);
```

Comment:
Any chance to make the duration time configurable? If I'm not mistaken there was even an issue about that once (not sure if it's still open).

Reply:
could be... but that should be discussed with the other ticket... it is still open, see #3126

Reply:
Based on this PR I've started to rewrite messages to be more like client-side, where delay is configurable.

Reply:
going to merge this... your PR can follow